deploy/deploy_volcengine.sh

```
@@ -406,6 +437,16 @@ else
   SMOKE_FAIL=1
 fi
 
+echo "  测试伦理台 API 路由是否挂载..."
+ETHICS_SMOKE=$($SSH_CMD "curl -s -o /dev/null -w '%{http_code}' 'http://127.0.0.1:8001/api/v1/ethics/researcher/messages' || true")
```

Shellcheck (0.11.0)

[warning] Quotes/backslashes in this variable will not be respected.

(SC2090)